features/support/object_model_pages/ivl/registration/ivl_family_information.rb

```diff
@@ -12,7 +12,11 @@ def self.individual_and_family_link
   end
 
   def self.continue_btn
-    '#btn_household_continue'
+    if enroll_registry[:bs4_consumer_flow].enabled?
+
```

Comment:
Should this if case be checking for a certain class instead of being empty?

Reply:
Guess this is more to confirm for all cases: Are we fine with having blank if blocks or do we want to change them to something like ```if !(enroll_registry[:bs4_consumer_flow].enabled?) ```

Reply:
I would go with the suggested `if !(enroll_registry[:bs4_consumer_flow].enabled?) `